packages/frontend/src/hooks/cartesianChartConfig/useCartesianChartConfig.ts

```diff
@@ -278,22 +278,6 @@ const useCartesianChartConfig = ({
         [dirtyLayout?.yField, updateAllGroupedSeries, pivotKeys],
     );
 
-    const setReferenceLines = useCallback((data) => {
```

Comment:
is this a leftover from somewhere?

Reply:
that was a function I initially implemented to set reference lines, but I ended up using a more generic `update series` but I forgot to remove it on a previous PR